nestkernel/node_collection.h

```diff
@@ -617,13 +626,13 @@ inline NodeIDTriple nc_const_iterator::operator*() const
   {
```

Comment:
I think inlining this operator makes not much sense given how complex it is.

Reply:
True, but not relevant for this PR. I've created an issue, #2203.